tests/ui_tools/test_popups.py

```diff
@@ -271,6 +275,14 @@ def mock_external_classes(
             return_value="Tue Mar 13 10:55 AM",
         )
 
+        mocker.patch.object(self.controller.model, "_all_users_by_id", _all_users_by_id)
```

Comment:
This line will change based on another comment, and being consistent is great.

So it's worth keeping this consistent like you have already, and the code above does patch the object, but more generally we can directly set values on the controller, since it's a Mock(). That's a lot less verbose than repeatedly patching.